submissions/armenter66/popUp/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>PopUp Armen</title>
+		<link rel="stylesheet" href="./style.css" />
+	</head>
+	<body>
+		<div class="container">
+			<header class="header">
+				<div class="header__wrapper">
+					<div class="header__logo">
+						<a href="./" class="header__link">
+							<img src="img/logo.jpg" alt="Avatar" class="header__logo-icon" />
```

Comment:
Your tab spacing size is very big. It is 2 or 4 spaces usually. It is hard read your code. 
Please, reformat it.